provider/resources.go

```diff
@@ -346,7 +346,11 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 
 	if _, err := awsbase.GetAwsConfig(context.Background(), config); err != nil {
 		return fmt.Errorf("unable to validate AWS credentials " +
```

Comment:
Perhaps I'm overthinking this but "validate" almost makes it sound like we're interfacing with a cert authority or something. Any objections to "Invalid credentials"?

In other news, I googled this error message, and it appears that both AWS and Azure use this wording, and I kinda hate it but I may be alone in my opinion.